docs/guide/blog.md

```diff
@@ -135,7 +135,13 @@ func (k msgServer) CreatePost(goCtx context.Context, msg *types.MsgCreatePost) (
 
 ## Add gRPC to the Module Handler
 
-In the `x/blog/module.go` file:
+First, in the `blog` directory, run the following command to generate client query codes:
```

Comment:
Thanks for the contribution! I'm not sure, however, that scaffolding a query belongs in this section. Right now it's in the Display Posts section and it makes sense because we modify the query code right after scaffolding.

Reply:
You are right it does not belong to this section, but if we do not run it here, we face the `x/blog/types/query.pb.gw.go` file is not generated and we see an error saying that `types.RegisterQueryHandlerClient does not exist` and so we cannot create a blog post. We need to somehow have this scaffolding before this section.